File: backoffice/FondaLogicLayer/Implementation/RestaurantCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace com.ds201625.fonda.Implementation
{
    public class RestaurantCategory : com.ds201625.fonda.Domain.RestaurantCategory
    {
        public RestaurantCategory()
        {

        }

        public RestaurantCategory InsertNewCategory(String nameCategory)
        {
            RestaurantCategory category = new RestaurantCategory();
            category.Name = nameCategory;

            return category;
        }

        public bool CategoryExist(String nameCategory)
        {
            List<String> _ListNameCategory;
            return true;
        }
    }
}

File: frontoffice/app/src/main/java/com/ds201625/fonda/domains/RestaurantCategory.java
package com.ds201625.fonda.domains;

/**
 * Created by Adri on 06/05/2016.
 */
public class RestaurantCategory {

    private String nameCategory;

    public RestaurantCategory () {
        super();
    }

    public RestaurantCategory (String nameCategory) {
        this.nameCategory = nameCategory;
    }

    public String getNameCategory() {
        return nameCategory;
    }

    public void setNameCategory(String nameCategory) {
        this.nameCategory = nameCategory;
    }

}
